Kitodo/pom.xml

```diff
@@ -767,15 +767,6 @@
                 <artifactId>maven-war-plugin</artifactId>
                 <version>3.2.2</version>
                 <configuration>
-                    <webResources>
-                        <resource>
-                            <directory>${parent.basedir}/OpacPica-Plugin/target</directory>
```

Comment:
Can this already be removed? Even when, I think this should be done in another pull Request.

Reply:
It no longer works without dependency on the UGH.

Doing thi in another pull request is going to be difficult, I'm afraid. I mean, the newly implemented validation (Pull Request #2151) is based on this branch. How can that still be merged then?

What exactly should I do?